docs/docs/.static/api.json

```diff
@@ -282,7 +282,8 @@
             "type": "integer",
             "format": "int64",
             "name": "max-depth",
-            "in": "query"
+            "in": "query",
+	    "required": true
```

Comment:
This file is auto-generated from the comments in the code. Please run `make sdk` to update it.

Reply:
Ok. Sure